2014-09-02 16:09:30 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'field list'
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `co...', Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(1012): DboSource->execute('INSERT INTO `co...')
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Model.php(1850): DboSource->create(Object(HsAauCoach), Array, Array)
#4 E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\UserController.php(175): Model->save(Array)
#5 [internal function]: UserController->registerHSCoach()
#6 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UserController), Array)
#7 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UserController), Object(CakeRequest))
#9 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-02 16:10:31 Error: Fatal Error (1): Call to a member function create() on a non-object in [E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\UserController.php, line 191]
2014-09-02 16:10:31 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 191)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 191, Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'E:\xampp2\htdoc...', 191, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-02 16:12:38 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'field list'
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `co...', Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(1012): DboSource->execute('INSERT INTO `co...')
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Model.php(1850): DboSource->create(Object(HsAauCoach), Array, Array)
#4 E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\UserController.php(175): Model->save(Array)
#5 [internal function]: UserController->registerHSCoach()
#6 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UserController), Array)
#7 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UserController), Object(CakeRequest))
#9 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-02 16:16:07 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'field list'
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `co...', Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(1012): DboSource->execute('INSERT INTO `co...')
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Model.php(1850): DboSource->create(Object(HsAauCoach), Array, Array)
#4 E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\UserController.php(175): Model->save(Array)
#5 [internal function]: UserController->registerHSCoach()
#6 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UserController), Array)
#7 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UserController), Object(CakeRequest))
#9 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-02 16:16:50 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'field list'
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `co...', Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(1012): DboSource->execute('INSERT INTO `co...')
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Model.php(1850): DboSource->create(Object(HsAauCoach), Array, Array)
#4 E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\UserController.php(177): Model->save(Array)
#5 [internal function]: UserController->registerHSCoach()
#6 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UserController), Array)
#7 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UserController), Object(CakeRequest))
#9 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-02 16:17:58 Error: Fatal Error (1): Call to a member function find() on a non-object in [E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\Component\EmailComponent.php, line 150]
2014-09-02 16:17:58 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 150)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 150, Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'E:\xampp2\htdoc...', 150, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-02 16:50:53 Error: Fatal Error (1): Maximum execution time of 30 seconds exceeded in [E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Utility\String.php, line 234]
2014-09-02 16:50:53 Error: [FatalErrorException] Maximum execution time of 30 seconds exceeded
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Maximum executi...', 'E:\xampp2\htdoc...', 234)
#1 [internal function]: ErrorHandler::handleError(1, 'Maximum executi...', 'E:\xampp2\htdoc...', 234, Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Maximum executi...', 'E:\xampp2\htdoc...', 234, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-02 17:27:20 Error: Fatal Error (1): Call to a member function create() on a non-object in [E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\UserController.php, line 192]
2014-09-02 17:27:20 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /collegeprospectnetwork/register-hs-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 192)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 192, Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'E:\xampp2\htdoc...', 192, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-02 19:13:06 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'CollegeCoach.username' in 'where clause'
Request URL: /collegeprospectnetwork/register-college-coach.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Athlete...', Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Athlete...', Array, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Athlete...', false)
#4 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Model.php(2921): DboSource->read(Object(Athlete), Array)
#5 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Model\Model.php(2893): Model->_readDataSource('first', Array)
#6 E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\UserController.php(223): Model->find('first', Array)
#7 [internal function]: UserController->registerCollegeCoach()
#8 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UserController), Array)
#9 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UserController), Object(CakeRequest))
#11 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-02 20:27:21 Error: [MissingControllerException] Controller class AdminController could not be found.
Exception Attributes: array (
  'class' => 'AdminController',
  'plugin' => NULL,
)
Request URL: /collegeprospectnetwork/Admin/login
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2014-09-02 20:27:41 Error: Fatal Error (1): Call to a member function find() on a non-object in [E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\HomeController.php, line 85]
2014-09-02 20:27:41 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /collegeprospectnetwork/login.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 85)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'E:\xampp2\htdoc...', 85, Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'E:\xampp2\htdoc...', 85, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-02 20:29:20 Error: Fatal Error (1): Call to undefined method SessionComponent::destory() in [E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\HomeController.php, line 208]
2014-09-02 20:29:20 Error: [FatalErrorException] Call to undefined method SessionComponent::destory()
Request URL: /collegeprospectnetwork/logout.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'E:\xampp2\htdoc...', 208)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'E:\xampp2\htdoc...', 208, Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'E:\xampp2\htdoc...', 208, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-02 20:29:58 Error: Fatal Error (1): Call to undefined method SessionComponent::destory() in [E:\xampp2\htdocs\collegeprospectnetwork\app\Controller\HomeController.php, line 210]
2014-09-02 20:29:58 Error: [FatalErrorException] Call to undefined method SessionComponent::destory()
Request URL: /collegeprospectnetwork/logout.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'E:\xampp2\htdoc...', 210)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'E:\xampp2\htdoc...', 210, Array)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'E:\xampp2\htdoc...', 210, Array)
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-02 20:31:57 Error: [MissingActionException] Action HomeController::forgetpassword() could not be found.
Exception Attributes: array (
  'controller' => 'HomeController',
  'action' => 'forgetpassword',
)
Request URL: /collegeprospectnetwork/Home/forgetpassword
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(HomeController), Object(CakeRequest))
#2 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-09-02 20:32:30 Error: [MissingViewException] View file "E:\xampp2\htdocs\collegeprospectnetwork\app\View\Home\forgot_password.phtml" is missing.
Exception Attributes: array (
  'file' => 'E:\\xampp2\\htdocs\\collegeprospectnetwork\\app\\View\\Home\\forgot_password.phtml',
)
Request URL: /collegeprospectnetwork/forgot-password.php
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\View\View.php(468): View->_getViewFileName(NULL)
#1 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Controller\Controller.php(954): View->render(NULL, NULL)
#2 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(198): Controller->render()
#3 E:\xampp2\htdocs\collegeprospectnetwork\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(HomeController), Object(CakeRequest))
#4 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-02 20:37:40 Error: [MissingControllerException] Controller class AdminController could not be found.
Exception Attributes: array (
  'class' => 'AdminController',
  'plugin' => NULL,
)
Request URL: /collegeprospectnetwork/Admin/forgetpassword
Stack Trace:
#0 E:\xampp2\htdocs\collegeprospectnetwork\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
